Template structure and calling change.

mynamavi opened the pull request:
Template structure and calling change.
Added Settings link in plugin directory
Added redirect to setting page after plugin activation.

Commit:
Very important Changes.
Template structure and calling change.
Added Settings link in plugin directory
Added redirect to setting page after plugin activation.

haydenyoung reviewed (changes requested):
What are the "Very important Changes."? Could you please review the message; either use the title from the card you are working on or provide a better summary of the changes. This is an public, open source project.

mynamavi commented:
In description, all the updates are described.
There is no cards related to update.

Commit:
Following updates done:
Template structure and calling change.
Added Settings link in plugin directory.
Added redirect to setting page after plugin activation.

Commit:
Merge branch 'master' of https://github.com/mynamavi/collectionpress

mynamavi commented:
changed the commit and pull title.
